### PR TITLE
Query people based on the chosen filter

### PR DESCRIPTION
**Pull request by elayabharath**
+ For now simply prints in console the number of people present

### CODE REVIEW

File: lib/model.dart [hunk 1]
```diff
@@ -14,6 +14,16 @@ class Person {
   String location;
   List<String> themes;
   int messageCount;
+
+  Person(this.age, this.age_category, this.gender, this.IDPStatus,
+      this.location, this.themes, this.messageCount);
+
+  factory Person.fromObj(Map<String, dynamic> obj) {
```
Comment (lukechurch) [inline lib/model.dart]: Where would I get one of these objects from? Is the expectation that this is a map that firebase would give me? In which case we might make this `Person.fromFirebaseMap`

File: lib/interactive.dart [hunk 2]
```diff
@@ -39,12 +41,18 @@ class Interactive {
...
+  void _updateFilterOption(String value) async {
     value = value == "null" ? null : value;
     _selected.updateOption(value);
+    await _loadPeople();
     _render();
   }
 
+  void _loadPeople() async {
+    var people = await fb.readPeople(_selected.filter, _selected.option);
+    print("Loaded ${people.length} people");
```
Comment (lukechurch) [inline lib/interactive.dart:53]: Could we factor this into a log statement pls

**Review by lukechurch — APPROVED** [review]

File: lib/model.dart (hunk 1 above)
Comment (elayabharath) [reply lib/model.dart]: Yes it is from firebase map, updated the name.

File: lib/interactive.dart (hunk 2 above)
Comment (elayabharath) [reply lib/interactive.dart:53]: Sure, will add it in the next PR

**Pull request merged**